static/docs/get-started/agenda.md

```diff
@@ -1,36 +1,35 @@
 # Agenda
 
-In the next few sections we will build a simple natural language processing
-(NLP) project from scratch. If you'd like to get the final result or have some
-issues along the way, you can download the fully reproducible
-[GitHub project](https://github.com/iterative/example-get-started) with:
+In the next few sections we'll build a simple natural language processing (NLP)
+project from scratch. If you'd like to get the final result or have any issues
+along the way, you can download the fully reproducible
+[GitHub project](https://github.com/iterative/example-get-started) by running:
 
 ```dvc
 $ git clone https://github.com/iterative/example-get-started
 ```
 
-Otherwise, bear with us and we will introduce the basic DVC concepts and get to
+Otherwise, bear with us and we'll introduce some basic DVC concepts to get to
 the same result together!
 
-The idea of the project is a simplified version of the
-[Tutorial](/doc/tutorials/deep). It explores the NLP problem of predicting tags
-for a given StackOverflow question. For example, we want a classifier that can
-predict posts about the Python language by tagging them `python`.
+The idea of the project is a simplified version of our
+[text classification tutorial](/doc/tutorials/deep). It explores the NLP problem
+of predicting tags for a given StackOverflow question. For example, we might
+want a classifier that can flag (or predict) posts about Python by tagging them
+with `python`.
 
 ![](/static/img/example-flow-2x.png)
 
-Do not let the NLP nature of the example discourage you from using DVC in other
-Data Science areas. There was no strong reason behind picking the NLP area. On
-contrary, DVC is designed to be agnostic of frameworks, languages, etc. If you
-have data files or datasets and/or you produce other data files, models,
+This is a natural language processing context, but NLP isn't the only area where
+DVC can help. DVC is designed to be agnostic of frameworks, languages, etc. If
+you _have_ data files or datasets and/or you _produce_ data files, models, or
```

Comment:
No need to add _italic_ formatting here. We don' really use that much and I'm not seeing the special importance of terms "have" and "produce". Thanks for the suggestion though!

Reply:
I see this is addressed now. Resolving